tools/node_modules/acorn/dist/acorn.js

```diff
@@ -2769,7 +2769,7 @@ pp$4.raise = function(pos, message) {
   message += " (" + loc.line + ":" + loc.column + ")";
   // XXX EMSCRIPTEN: add a quote of the error text, point to where
   // https://github.com/acornjs/acorn/pull/793
-  var lines = this.input.split('\n');
+  var lines = this.input.split(lineBreak);
   message = message + '\n' + lines[loc.line - 1] + '\n';
```

Comment:
Note that this `\n` goes to the error message string, which is later displayed by Node exception handler. Hence, it's a usual string, not bytes, newlines are transformed automatically.